8_visualize.yml

```diff
@@ -0,0 +1,90 @@
+target_default: 8_visualize
+
+packages:
+  - sf
+  - tidyverse
+  - lwgeom
+  - smoothr
+  - igraph
+  - purrr
+
+sources:
+  - 8_visualize/src/visualization_functions.R
+  - 8_visualize/src/freq_dist_functions.R
+
+targets:
+  8_visualize:
+    depends:
+
+  # map of network and subset
+  8_visualize/out/map_network_subset.png:
+    command: plot_subnet(
+      out_file = target_name,
+      subnet_ind = '9_collaborator_data/out/network_subset.rds.ind', 
+      network_ind = '1_network/out/network.rds.ind', 
+      summary_ind = '9_collaborator_data/out/delaware_sites_summary.rds.ind')
```

Comment:
I think I modified this intentionally, because I was confused by why there were so many missing "links" in the network. I think this is because the 1-10 is gray, so is graying out the network and blending in with the background. I will modify to the original, but I'm fairly confident the underlying data are the same. But will verify that it matches your above old plot.

Reply:
Hmm, maybe...but I was thinking there really shouldn't be points so far from the reaches after filtering to those reaches that we really believe are ON those reaches (filtering by bird distance) and sufficiently close to an outlet (filtering by fish distance).

Reply:
Ok, there was a couple of things going on. There was the color issue, which I updated, but I was also not filtering out the sites that were excluded because their match distance (fish and bird) was too large. Now looks like this: 
![image](https://user-images.githubusercontent.com/15788176/73686002-d029fd80-468c-11ea-8302-d9722187e3ed.png)

Reply:
😍